packages/core/src/Components/Button/UI/FoLoadingButton.vue

```
@@ -23,37 +23,29 @@
 </template>
 
 <script setup lang="ts">
-import type { ButtonProps }        from '@/Components/Button/Types/Button';
+import type { LoadingButtonProps } from '@/Components/Button/Types/Button';
 import type { LoadingProps }       from '@/Components/Loading/Types/Loading';
 import type { HorizontalPosition } from '@/Shared/Utils';
-import type { VNode }              from 'vue';
+import type { Slot }               from 'vue';
```

ESLint

[error] Expected "vue" (type) to come before "@/Shared/Utils" (internal-type).

(perfectionist/sort-imports)